Automattic/vip-go-mu-plugins
# Disable per-site metrics for the time being

## Description

For the time being, need to disable individual site StatsD metrics.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Pull down PR
1. Ensure nothing with Search is broken
